dev_guide/builds.adoc

```diff
@@ -88,8 +110,11 @@ image tag or the source code changes:
 *ruby-sample-build*.
 <2> You can specify a list of link:#build-triggers[triggers], which cause a new
 build to be created.
-<3> The `*source*` section defines the source code repository location. You can
-provide additional options, such as `*sourceSecret*` or `*contextDir*` here.
+<3> The `*source*` section defines the source of the build. The source type
+determines the primary source of input, and can be either `*Git*`, to point to a
+code repository location, `*Dockerfile*`, to build from an inline Dockerfile, or
+`*Binary*`, to accept binary payloads. It is possible to have multiple sources
+at once, refer to the documentation for each source type for details.
```

Comment:
It is a note, because even though "It is possible to have multiple sources at once", not all combinations of sources and strategy are valid.